src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -286,9 +284,9 @@
         WorkflowRun b2 = r.assertBuildStatusSuccess(p.scheduleBuild2(0));
 
         // Verify that we're seeing warnings due to running 'properties' in a non-multibranch job.
-        r.assertLogContains("WARNING: The 'properties' step will remove all 'JobProperty's currently configured in this job, either from the UI or from an earlier 'properties' step.", b2);
+        r.assertLogContains(Messages.JobPropertyStep__could_remove_warning(), b2);
         // Verify that we *are* seeing warnings for removing the triggers property.
-        r.assertLogContains("WARNING: Removing existing job property 'Build triggers'", b2);
+        r.assertLogContains(Messages.JobPropertyStep__removed_property_warning("Build triggers"), b2);
```

Comment:
:ant: can use `r.jenkins.getDescriptorByType(Whatever.DescriptorImpl.class).getDisplayName()` to avoid hard-coding.